src/src/secret.py

```diff
@@ -1,8 +1,10 @@
 def getGithubKey():
-    return 'GHKEYXXX'
+    return '376d679b554ae9a9d474'
+    # return 'GHKEYXXX'
```

Comment:
Remove this file from the PR - it includes sensitive data

Reply:
I have corrected this.